runtime/src/main.cpp

```diff
@@ -759,30 +759,32 @@ int main(int argc, char** argv, MPI_Comm comm, int shrMemPID)
 
       for (auto const& p : bodyPositions)
       {
-      	mass = p.w;
-      	center_of_mass.x += mass * p.x;
-      	center_of_mass.y += mass * p.y;
-      	center_of_mass.z += mass * p.z;
-      	center_of_mass.w += mass;
+        mass = p.w;
+        center_of_mass.x += mass * p.x;
+        center_of_mass.y += mass * p.y;
+        center_of_mass.z += mass * p.z;
+        center_of_mass.w += mass;
       }
 
       center_of_mass.x /= center_of_mass.w;
       center_of_mass.y /= center_of_mass.w;
       center_of_mass.z /= center_of_mass.w;
 
-      // get total velocity
-      real4 total_velocity = make_real4(0.0, 0.0, 0.0, 0.0);
+      // get center-of-mass velocity
+      real4 center_of_mass_velocity = make_real4(0.0, 0.0, 0.0, 0.0);
 
-      for (auto const& v : bodyVelocities)
+      for (int i = 0; i < bodyVelocities.size(); i++)
```

Comment:
size_t should be used for i to avoid compiler warning

Reply:
Ok, fixed this.